src/core/constants/appConfig.ts

```diff
@@ -110,7 +110,7 @@ const appConfig = {
 		NL_TOGGLE: "nl"
 	},
 	MAP_RESOURCES: {
-		IMPERIAL_COUNTRIES: ["US", "GB", "LR", "MM"],
+		IMPERIAL_COUNTRIES: ["US", "LR", "MM"],
```

Comment:
GB should still be a country that defaults to Imperial in this context.

From here: https://en.wikipedia.org/wiki/Metrication_in_the_United_Kingdom#:~:text=The%20United%20Kingdom%2C%20through%20voluntary,km/h)%20units%20displayed.

"Apart from the sale of fuel, which was metricated in the 1980s, motorists have seen little metrication. [Speedometers](https://en.wikipedia.org/wiki/Speedometer) and mandatory information on car advertisements such as [fuel consumption](https://en.wikipedia.org/wiki/Fuel_efficiency) are given in both metric and imperial units. The [1994 TSRGD](https://en.wikipedia.org/wiki/Traffic_Signs_Regulations_and_General_Directions) permitted the use of metric units alongside imperial units for width and height warning signs[[91]](https://en.wikipedia.org/wiki/Metrication_in_the_United_Kingdom#cite_note-TSRGD1994-92) and dual metric/imperial signs became mandatory from March 2015.[[92]](https://en.wikipedia.org/wiki/Metrication_in_the_United_Kingdom#cite_note-:0-93) Distances and speed restrictions are shown only in imperial units.[[91]](https://en.wikipedia.org/wiki/Metrication_in_the_United_Kingdom#cite_note-TSRGD1994-92)"

Since distance/speed signs are either in dual-unit or only imperial, but never only metric, I believe imperial would be the correct default.

Reply:
resolved